### ISSUE TITLE
[1.01.0024] Visual Studio Stops responding when refreshing attach to process list after detaching and re-attaching device

### ISSUE THREAD

**Issue by zadmarbella** [blackberry/VSPlugin#341]
Preq: a project loaded on the device.

```
1. Debug > Attach To Process
2. Select Transport BlackBerry
3. Open attached project to run that process on device/simulator
4. Select Qualifier Device/Simulator: <IP Address>
5. Disconnect the device
6. Reconnect the device
7. Hit Refresh 
8. Click yes on the Attempting to connect to device/simulator dialog 
```

Behaviour:
Attach to process window stops responding and vs becomes unusable.

Note: it looks like waiting long enough prompts the unable to connect but attempting to interact with vs will prompt the notification that VS is not responsive.

Workaround: 
Wait until device is fully connected

**Comment by guarnold** [comment]
Now, the current behavior after the timeout (7 to 8 seconds) is to present a message indicating that the device is not connected and that the current task will be cancelled. The user must connect the device and then restart the same task, if he wants.

**Comment by guarnold** [comment]
Available in build 1.01.0026

**Comment by zadmarbella** [comment]
[1.01.0041] verified win8 vs 2012 win 7 vs2010
